packages/backend-deployer/src/cdk_error_mapper.ts

```diff
@@ -372,6 +359,28 @@ export class CdkErrorMapper {
       errorName: 'SecretNotSetError',
       classification: 'ERROR',
     },
+    {
+      errorRegex: /The code contains one or more errors/,
```

Comment:
1. Few internal/external searches revealed this is AppSync specific only. The error from hotswap is `Error: [31m  [1mamplify-redacted-stack-11111111111[22m failed: BadRequestException: The code contains one or more errors.[39m`
2. No, unfortunately the error messages have no distinguishable information

Reply:
Ok.
Regarding 1.
Would `/BadRequestException: The code contains one or more errors/` ? So that at least there's clear signal that message came from service?

Reply:
That won't work either, the CFN error message is `Deployment failed: Error: The stack named amplify-redacted-stack-1111111 failed to deploy: UPDATE_ROLLBACK_COMPLETE: Resource handler returned message: The code contains one or more errors. (Service: AppSync, Status Code: 400,...`

We can do `/BadRequestException: The code contains one or more errors|/The code contains one or more errors/.*AppSync/` if we want to be more specific

Reply:
Updated Regex.

Reply:
ty